pkg/utils/apiutil/apiutil.go

```diff
@@ -336,6 +336,22 @@ func ParseKey(name string, input map[string]any) ([]byte, string, error) {
 	return returned, rawKey, nil
 }
 
+// ParseHexKeys decodes hexadecimal keys to bytes if the format is "hex".
+func ParseHexKeys(format string, keys []string) (hexStrings []string, err error) {
```

Comment:
```suggestion
func ParseHexKeys(format string, keys []string) (hexBytes [][]byte, err error) {
```

returning bytes can reduce some converting

Reply:
changed in https://github.com/tikv/pd/pull/8262/commits/3f515a6d39dad824fb9499492988c6dd7af65dd4